From a68a8d051b476ceae87e0490001b7c60ac5ea942 Mon Sep 17 00:00:00 2001
From: Ritesh Raj Sarraf <rrs@debian.org>
Date: Wed, 17 Apr 2013 12:11:01 +0530
Subject: [PATCH] Don't call killall with the -g argument.

It kills the entire group of processes
https://bugs.gentoo.org/show_bug.cgi?id=463944

Thanks: Murray Campbell
---
 usr/sbin/laptop_mode | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/sbin/laptop_mode b/usr/sbin/laptop_mode
index 74a129c..cfd99c1 100755
--- a/usr/sbin/laptop_mode
+++ b/usr/sbin/laptop_mode
@@ -1130,7 +1130,7 @@ if [ x$ENABLE_BATTERY_LEVEL_POLLING = x1 ] && [ x$BLACKLIST_IN_FLOCK = x1 ]; the
 		log "VERBOSE" "On AC, stopping the polling daemon."
 
 		# In AC mode we disable the polling daemon.
-		killall -g -q lm-polling-daemon
+		killall -q lm-polling-daemon
 	elif [ x$ON_AC = x0 ]; then
 		exec 7>$LMT_BATTPOLL_LOCK;
 		if $FLOCK -n -x -w 1 7; then
@@ -1149,7 +1149,7 @@ if [ x$ENABLE_BATTERY_LEVEL_POLLING = x1 ] && [ x$BLACKLIST_IN_FLOCK = x1 ]; the
 		# should be killed.
 		if [ x$INITSCRIPT_STOP = x1 ]; then
 			log "VERBOSE" "On Battery, but init script stop is called. Killing lm-polling-daemon"
-			killall -g -q lm-polling-daemon
+			killall -q lm-polling-daemon
 		fi
 	else
 		log "ERR" "Unknown ON_AC state: $ON_AC";
-- 
1.8.1.6

